deployments/arbitrum/usdc/configuration.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "Compound USDC",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0xff970a61a04b1ca14834a43f5de4533ebddb5cc8",
+  "baseTokenPriceFeed": "0x50834F3163758fcC1Df9973b6e91f0F0F0434aD3",
+  "borrowMin": "100e6",
+  "storeFrontPriceFactor": 0.5,
+  "targetReserves": "5000000e6",
+  "rates": {
+    "supplyKink": 0.8,
+    "supplySlopeLow": 0.0325,
+    "supplySlopeHigh": 0.4,
+    "supplyBase": 0,
+    "borrowKink": 0.8,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 0.25,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "0e15",
+    "baseBorrowSpeed": "0e15",
+    "baseMinForRewards": "10000e6"
+  },
+  "assets": {
+    "LINK": {
```

Comment:
Not final assets or configuration parameters; just needed some values to get scenarios passing.

Added LINK because WBTC is difficult to source in large quantities, and using WETH as `$asset0` is incompatible with the Bulker scenarios.